crates/cranelift/src/compiler.rs

```diff
@@ -942,6 +942,7 @@ impl Compiler {
         let slot = builder.func.create_sized_stack_slot(ir::StackSlotData::new(
             ir::StackSlotKind::ExplicitSlot,
             values_vec_byte_size,
+            0,
```

Comment:
Can this be set to 4? The ValRaw bits in Wasmtime require a 16-byte alignment

Reply:
Done!